src/cljs/aikakonematka/util.cljs

```diff
@@ -222,3 +223,7 @@
       (:play-time-text derefed-state)
       (str play-time-in-sec))
     (swap! game-state assoc :play-time play-time-in-sec)))
+
+(defn update-music-notes [music-notes]
+  (println "music notes : " music-notes)
+  (swap! game-state assoc :music-notes music-notes))
```

Comment:
it is not adding new note (`music-notes`) to the collection but it's replacing the whole thing with new value